### DISCUSSION TITLE
S1 Pro and Linear Advance

### DISCUSSION THREAD

**Discussion by thunderpipesusa** [mriscoc/Ender3V2S1#613] [Q&A]
Currently running the T13 experimental on my ender 3 s1 pro and it has been great.  I dont necessarily print high temps on this printer though so I prob just want to move to the newest release for the f4 chip using UBL

However, I see an experimental now with LA

Question is does that experimental also contain UBL?

I know certain things need to be disabled to make room but is that already done in that experimental latest release version?

I have no idea how to compile anything.  I guess I could learn but definitely would appreciate any links or info on this question.  I want the new features available in the newest release for the f4 UBL but would like to try the LA to speed up my machine. 

Also, Is there a screen firmware update to use the new features of the latest releases?

**Comment by mriscoc** [comment]
> Question is does that experimental also contain UBL?  

No, only files with UBL in the filename have it.

F4 boards do not have enough program memory to try many things at the same time.

**Reply by thunderpipesusa** [reply]
thanks for the response, is there a tutorial on how to use the configurator or a step by step video for noobs :) thanks in advance.

i have installed visual studio code and can see the configurator after downloading the repository but it wont run

I have a list of things to try and disable to see if i can get ubl and la at the same time. 
disable...
resume on power loss maybe some error coded if using a terminal.... 

I have the T13 Thermistor as well  omg im confusing myself....lol

OK so do you think you would be able to compile something with UBL, LA for the f4 with T13

with the latest marlin like the latest releases you have.

If so I would happily contribute.  

If I cant get anything going that route, is there a T13 version with UBL?

**Reply by mriscoc** [reply]
Currently, there aren't videos about the Configurator. 
The Configurator needs a proper Python installation to be executed.

**Comment by thunderpipesusa** [comment]
I have python installed already on windows.  Still won’t start.Ok how about this.I have the t13 thermistor.  S1 pro.Do I need to use the experimental t13 only or can I use the f4 ubl or the experimental LA to get the thermistor to keep its temp balanced.  As stated I don’t use anything more than 250 -260 temp range at maxSent from my iPhoneOn Jan 5, 2023, at 12:08 AM, Miguel Risco-Castillo ***@***.***> wrote:﻿
Currently, there aren't videos about the Configurator.
The Configurator needs a proper Python installation to be executed.

**Reply by mriscoc** [reply]
The Sprite Extruder Pro uses the thermistor type 13.

**Comment by thunderpipesusa** [comment]
Ok thanks for the info. I understand the sprite uses T13  I am asking if
you know if I "have" to use the T13 or if I can successfully use the
release you have of the f4 with UBL   or the release you have with LA
(experimental) without having any abnormal hot end temperature fluctuations

A Facebook friend has compiled a firmware from your repository that is for
f4, has t13, ubl, and LA. I am not sure what he took out to make it work.
Is there a way for me to get you the bin file to look over for me? I will
try to attach here but I don’t know if it will upload to you correctly.



On Jan 5, 2023, at 6:49 AM, Miguel Risco-Castillo ***@***.***>
wrote:

﻿

The Sprite Extruder Pro uses the thermistor type 13.

—
Reply to this email directly, view it on GitHub
<https://github.com/mriscoc/Ender3V2S1/discussions/613#discussioncomment-4601704>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/A4IDQWSLZIG7W7DFPDD7LBLWQ3GNBANCNFSM6AAAAAATQ7ROIA>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>
